lang/en.yml

```diff
@@ -149,7 +187,7 @@ en:
     HelpChars: ' Special characters are automatically converted or removed.'
     OK: OK
   SilverStripe\CMS\Model\RedirectorPage:
-    CLASS_DESCRIPTION: 'Redirects to an internal page or an external URL'
+    CLASS_DESCRIPTION: 'Redirects requests to another location'
```

Comment:
Why is this changing?

Reply:
Beats me. The PHP had that value in CMS 5 as well, so the question really is "why wasn't this in the yaml before?"
The text collection is correctly setting this to the value in PHP-land.

If you want I can change the PHP to match this value, but either way the PHP and YAML need to match, and both have been at these values for quite a while from what I can see.

Reply:
Will hold off updating the failing behat test pending your decision

Reply:
OK keep as is in this PR

Reply:
Leave as is per https://github.com/silverstripe/silverstripe-cms/pull/3080/files#r2094642942